tests/PolySharp.PolySharpUseTypeAliasForUnmanagedCallersOnlyAttribute.Tests/PolySharp.PolySharpUseTypeAliasForUnmanagedCallersOnlyAttribute.Tests.csproj

```diff
@@ -3,12 +3,12 @@
   <PropertyGroup>
     <TargetFramework>net472</TargetFramework>
     <PolySharpIncludeRuntimeSupportedAttributes>true</PolySharpIncludeRuntimeSupportedAttributes>
-    <PolySharpUseTypeAliasForUnmanagedCallersOnlyAttribute>true</PolySharpUseTypeAliasForUnmanagedCallersOnlyAttribute>
+    <PolySharpUseInteropServices2NamespaceForUnmanagedCallersOnlyAttribute>true</PolySharpUseInteropServices2NamespaceForUnmanagedCallersOnlyAttribute>  
   </PropertyGroup>
 
   <ItemGroup>
     <CompilerVisibleProperty Include="PolySharpIncludeRuntimeSupportedAttributes" />
-    <CompilerVisibleProperty Include="PolySharpUseTypeAliasForUnmanagedCallersOnlyAttribute" />
+    <CompilerVisibleProperty Include="PolySharpUseInteropServices2NamespaceForUnmanagedCallersOnlyAttribute" />
   </ItemGroup>
 
   <ItemGroup>
```

Comment:
You should add a unit test to check that the  is being declared and/or forwarded correctly, here's the code I'm using:
```csharp
[Fact]
public void CheckMember()
{
	var t = typeof(SomeTypeWithInitProperty);
	var m = t.GetMethod("set_SomeInitProperty", BindingFlags.Public | BindingFlags.Instance);
	var modifiers = (optional: m.ReturnParameter.GetOptionalCustomModifiers(), required: m.ReturnParameter.GetRequiredCustomModifiers()); //get list of modifiers
	Assert.Equal(new Type[0], modifiers.optional); //check there's not modopts
	Assert.Single(modifiers.required); //check there's 1 modreq
	Assert.Equal("System.Runtime.CompilerServices.IsExternalInit", modifiers.required[0].FullName); //check that it's what we expect
	const string expectedAssemblyName = "yourTestLibraryName";
#if !NET5_0_OR_GREATER
	//Defined in expectedAssemblyName assembly
	Assert.Equal(expectedAssemblyName, modifiers.required[0].Assembly.GetName().Name); //check it's in the assembly we expect
#else
	//Type forwarded to the existing definition
	var ex = typeof(IsExternalInit).Assembly.GetName().Name; //get the expected actual assembly name
	Assert.NotEqual(expectedAssemblyName, ex); //check we're not picking up one from expectedAssemblyName on accident due to it being public or something
	Assert.Equal(ex, modifiers.required[0].Assembly.GetName().Name); //check the one we have on our type is the one we expect
	Assert.Equal(ex, t.Assembly.GetType("System.Runtime.CompilerServices.IsExternalInit").Assembly.GetName().Name); //check that type forwarding is working as expected
#endif
}
```

Reply:
Done, thank you! 🙂